dna/zomes/projects/src/project/member/validate.rs

```diff
@@ -4,6 +4,7 @@ use crate::project::{
 use hdk::prelude::*;
 
 #[hdk_extern]
+/// Creates allowed if the member `address` matches the author of the entry
```

Comment:
👍 
this data type declares themselves to be a member of the project. we do a quick and easy lookup of these off an anchor to check who's in